jenkins/groovy/release-haproxy/Jenkinsfile

```diff
@@ -131,13 +131,5 @@ pipeline {				//indicate the job is written in Declarative Pipeline
                 }
             }
         }
-        // trigger rotation of load balancers to trigger reconfigure step using new branches
```

Comment:
This one actually needs it, as release-haproxy retags the instances and now they need to be reconfigured.